alpha/src/components/organisms/category/CategoryList.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Link, useLocation } from "react-router-dom";
+import { useRecoilValue } from "recoil";
+import styled from "styled-components";
+import { categoryState } from "../../../store/category";
+
+const CategoryList = () => {
+  const category = useRecoilValue(categoryState);
+  const loca = useLocation();
+  const categoryId = parseInt(loca.pathname.split("/")[2]);
+
+  return (
+    <CategoryListBlock>
+      <h2>커뮤니티</h2>
+      {category.map(({ id, name }, idx) => {
+        if (id === categoryId)
+          return (
+            <Link to={`/category/${id}`} className="nav-links" key={idx}>
+              <li className="curCategory">{name}</li>
+            </Link>
+          );
+        else
+          return (
+            <Link to={`/category/${id}`} className="nav-links" key={idx}>
+              <li>{name}</li>
+            </Link>
+          );
```

Comment:
```suggestion
          return (
            <Link to={`/category/${id}`} className="nav-links" key={idx}>
              <li className={id === categoryID ? "curCategory" : ""}>{name}</li>
            </Link>
          );
```

li의 클래스명 지정하는 부분을 `className={id === categoryID ? "curCategory" : ""}` 으로 수정하시면 나머지 부분은 아래의 else에서 반환하는 Link 컴포넌트랑 생김새가 같으니까 조건문이 필요없어보이기도하네용
사실 취향차이라 이대로 두셔도 크게 상관은 없을것같읍니다